backend/tursu_admin/views.py

```diff
@@ -64,3 +65,30 @@ def verify_product(request):
     product.is_verified = True
     product.save()
     return HttpResponse("Successfully verified product")
+
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes((IsAuthenticated,))
+@api_view(['POST'])
+def ban_user(request):
```

Comment:
1. Admin authentication should be done.
2. At least one parameter to find the respective customer or vendor should be taken. My suggestion would be to take email or username which are common in both user types and then the retrieval can be done in the function.

Reply:
Necessary changes are made.